### PR TITLE
doc in book standard-types-and-funcitons string

### PR DESCRIPTION
**Pull request by frehberg**

### CODE REVIEW

File: book/src/standard-types-and-functions.md [hunk 1]
```diff
@@ -17,6 +17,53 @@ When compiling an expression, the compiler automatically inserts a small prelude
 
 Gluon has support for cooperative threading and communication between them through the `Thread` and `Sender`/`Receiver` types.
 
+### Strings
+
+`String` is a built-in data type. The module `std.string` provides the infix operatoin `++`, concatenating two strings. The operation `show` converts the Int to a printable String.
+```
+let io @ { ? } = import! std.io
+let string @ { (++), len } = import! std.string
```
Comment (Marwes) [inline book/src/standard-types-and-functions.md:25]: `++` is already in the prelude so you do not need to import it again https://github.com/gluon-lang/gluon/blob/25d949b8836d0cd28adc027f8d0e4e20a102aca1/src/lib.rs#L875

File: book/src/standard-types-and-functions.md [hunk 2]
```diff
@@ -17,6 +17,53 @@ When compiling an expression, the compiler automatically inserts a small prelude
...
+let message : String = "Hello" ++ " " ++ "World" ++ "!"
+let prompt s = "<" ++   show (len s) ++ "> "
+io.println ( (prompt message) ++ message )
+```
+The output will be
+```
+<12> Hello World!
+```
+
+The module `std.string` is providing the following operations
```
Comment (Marwes) [inline book/src/standard-types-and-functions.md:36]: Could link to the API reference directly instead https://gluon-lang.org/doc/crates_io/std/std/string.html

File: book/src/standard-types-and-functions.md (hunk 1 above)
Comment (frehberg) [reply book/src/standard-types-and-functions.md]: done

File: book/src/standard-types-and-functions.md (hunk 2 above)
Comment (frehberg) [reply book/src/standard-types-and-functions.md]: done

**Comment by frehberg** [comment]
Hi, wondering when this is merged? Is there any TODO open?

**Pull request merged**